packages/headless/src/app/engine.ts

```diff
@@ -93,6 +94,12 @@ export interface EngineOptions<Reducers extends ReducersMapObject>
    * [Redux documentation on reducers.](https://redux.js.org/glossary#reducer)
    */
   reducers: Reducers;
+  /**
+   * An optional cross slice (or root) reducer that can be combined with the slice reducers.
+   *
+   * [Redux documentation on root reducers.](https://redux.js.org/usage/structuring-reducers/beyond-combinereducers)
+   */
+  crossSlice?: Reducer;
```

Comment:
Picky: i'd keep the name reducer, rather than slice for the option. `crossReducer` `rootReducer` is good. We only use slice in the file name, but reducer is the correct term. A slice is the combination or reducer & it's related actions